test/test/TestREPL.scala

```diff
@@ -38,7 +38,8 @@ class TestREPL(script: String) extends REPL {
     out.close()
     val printed = out.toString
     val transcript = printed.drop(printed.indexOf(config.prompt))
-    if (transcript.toString != script) {
+    val transcriptNoColors = transcript.toString.replaceAll("\u001B\\[[;\\d]*m", "")
```

Comment:
the scala REPL has `-Dscala.color`: https://github.com/scala/scala/blob/99a82be91cbb85239f70508f6695c6b21fd3558c/src/repl/scala/tools/nsc/interpreter/ReplProps.scala#L20